src/CameraSensor.cc

```diff
@@ -271,6 +281,14 @@ bool CameraSensor::Load(const sdf::Sensor &_sdf)
   igndbg << "Camera images for [" << this->Name() << "] advertised on ["
          << this->Topic() << "]" << std::endl;
 
+  if (_sdf.CameraSensor()->Triggered() == true)
+  {
+    this->dataPtr->node.Subscribe(this->Topic() + "/trigger",
```

Comment:
this topic suffix is called `image_trigger` in gazebo_ros_pkgs

https://github.com/ros-simulation/gazebo_ros_pkgs/blob/ros2/gazebo_plugins/src/gazebo_ros_camera.cpp#L272

I think we can maintain consistency with that topic name